src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ArrayConverter.cs

```diff
@@ -33,7 +32,81 @@ public override object ConvertTo(ITypeDescriptorContext context, CultureInfo cul
 
             return base.ConvertTo(context, culture, value, destinationType);
         }
-    }
-}
 
+#if !NETSTANDARD10
```

Comment:
Could we do something other than using such ifdefs?  For example, could we include these additional functions in a *.NetStandard.cs partial class file, or something like that?